scipy/optimize/_dual_annealing.py

```diff
@@ -256,9 +256,13 @@ def __init__(self, acceptance_param, visit_dist, func_wrapper,
 
     def accept_reject(self, j, e, x_visit):
         r = self._rand_gen.uniform()
-        pqv_temp = (self.acceptance_param - 1.0) * (
+        pqv_temp_old = (self.acceptance_param - 1.0) * (
             e - self.energy_state.current_energy) / (
                 self.temperature_step + 1.)
+        assert pqv_temp_old <= 0
```

Comment:
This code will be deleted before I submit it's just here to demonstrate how absurd the old code is.